src/server/game/Spells/SpellInfo.cpp

```diff
@@ -2061,6 +2061,7 @@ AuraStateType SpellInfo::LoadAuraState() const
         case 9806:  // Phantom Strike
         case 35325: // Glowing Blood
         case 16498: // Faerie Fire
+        case 6950: // Faerie Fire - Razormane Thornweaver
```

Comment:
Comment not needed as it is the same as above.

Also this spell is used by more NPCs so "Razormane Thornweaver" not needed

Reply:
Correct 🐱